AcademicFlow.Domain.Contracts/IServices/ICourseService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using AcademicFlow.Domain.Contracts.Entities;
+
+namespace AcademicFlow.Domain.Contracts.IServices
+{
+    public interface ICourseService
+    {
+        int? AddCourse(Course course);
```

Comment:
why the method is not async? look at other services, it should be async and return Task